frontend/src/widgets/HorizontalTableWidget.scss

```diff
@@ -115,4 +103,33 @@
 
 .smarthub-horizontal-table-widget tfoot td {
   font-weight: bold;
-}
\ No newline at end of file
+}
+
+// First footer cell sticky.
+.smarthub-horizontal-table-widget tfoot td:first-child {
+  left: 0;
+  position: sticky;
+  z-index: 1;
+  min-width: 64px;
+}
+
+// Second footer cell sticky.
+.smarthub-horizontal-table-widget tfoot td:nth-child(2) {
+  position: sticky;
+  z-index: 1;
+  max-width: 330px;
+  left: 44px;
+}
+
+// Make sure first column is always sticky.
+.smarthub-horizontal-table-widget .smarthub-horizontal-table-first-column {
+  position: sticky;
+  z-index: 1;
+  max-width: 330px;
+}
+
+// Keep the border regardless of the size.
+.smarthub-horizontal-table-widget
+.smarthub-horizontal-table-first-column-border {
+box-shadow: inset 0 0 0 black, inset -1px 0 0 black;
+}
```

Comment:
at the design/dev meeting they said they didn't want this, did they change their mind?

Reply:
They don't want a border on the first column?